src/pudl/package_data/settings/etl_fast.yml

```diff
@@ -42,7 +42,7 @@ datasets:
       years: [2020, 2023]
     eia860:
       years: [2020, 2022]
-      eia860m: true
+      eia860m_year_months: ["2023-06"]
```

Comment:
Does it make sense produce a change log from one month of data? Should we load the last year or all of the data?

Reply:
yea this is a fair question.. One month is definitely not a change log. I think it would take a while to do a full year so maybe like a lot of the other fast settings we feed this two partitions. it'll be kinda a silly lil changelog but i think that is okay for the fast CI

Reply:
Yeah!  I think a two-month changelog is sufficient for testing out any logic that expects multiple months of data.